tests/v2.test.js

```diff
@@ -0,0 +1,338 @@
+import expect from 'expect';
+import fs from 'promise-fs';
+import path from 'path';
+import request from 'request-promise-native';
+import uuid from 'uuid/v1';
+
+const { URLSearchParams } = require('url');
+
+const api = request.defaults({
+  baseUrl: 'https://api.fastly.com/service/6lQpb69X5Qt9YVt2OIMiGG/',
+  headers: {
+    Accept: 'application/json',
+    'Fastly-Key': process.env.FASTLY_API_KEY,
+  },
+});
+
+const domain = 'iotest--iiif.elifesciences.org';
```

Comment:
Reusing my original IO experimentation domain, should probably change to something clearer.